cfme/tests/services/test_provision_stack.py

```diff
@@ -174,7 +174,7 @@ def create_template(setup_provider, provider, dialog):
     yield dialog
 
 
-@pytest.mark.meta(blockers=[BZ(1340570, forced_streams=["5.6", "upstream"])])
+@pytest.mark.meta(blockers=[BZ(1346075, forced_streams=["5.5", "5.6", "upstream"])])
```

Comment:
Does this affect AWS providers as well? If not, it should probably block only for Openstack: `@pytest.mark.meta(blockers=[BZ(1346075, forced_streams=["5.5", "5.6", "upstream"]), unblock=lambda provider: provider.type != 'openstack'])`